hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -247,12 +241,29 @@ public void testReplaceTxnBuilder() throws Exception {
     }
   }
 
+  @Test
+  public void testCreateTableWithOwner() throws Exception {
+    Schema schema = getTestSchema();
+    PartitionSpec spec = PartitionSpec.builderFor(schema).bucket("data", 16).build();
+    TableIdentifier tableIdent = TableIdentifier.of(DB_NAME, "tbl");
+    String location = temp.newFolder("tbl").toString();
+    String owner = "some_owner";
+    ImmutableMap<String, String> properties =
```

Comment:
Does this also write the username to the HMS table property list?
Is this what we expect?

Reply:
> Does this also write the username to the HMS table property list? Is this what we expect?

Good point! This was also added ad an HMS table property but I think it's not needed. I changed the code to only set the owner of the table but not to add the owner field as a table property to HMS.